src/rpc/net_type.h

```diff
@@ -27,6 +27,8 @@ struct RPCBase {
 };
 
 struct RPCSender : RPCBase {
+  static constexpr int kMaxTryCount = 1024;
```

Comment:
Consider remove this. You are passing the max retry times from Python to C++ anyway.

Reply:
I assign a default value for `max_try_times` in `ConnectReceiverFinalize` on purpose. As it's in base class and `max_try_times` may not be passed in some cases such as `//src/graph/network.cc`.

Reply:
My concern is that there are two constants: one coded here and another from environment variable. People may forget to change them together in the future. It's better to have only one version.

Reply:
ok. let me remove it.